test/functional/test_site/index.md

```diff
@@ -219,3 +219,9 @@ tags: ["tag-frontmatter-shown", "tag-included-file", "+tag-exp*", "-tag-exp-hidd
 <div id="test-markbind-plugin">
   Markbind Plugin Pre-render Placeholder
 </div>
+
+# Test search indexing
+
+## Level 2 header with noindex attribute should not be indexed {.noindex}
+
+###### Level 6 header with index attribute should be indexed {.index}
```

Comment:
I think it would be clearer if you say that the heading level is inside/outside the `headingSearchIndex`, so that in case this fails in the future (failing may be due to changed `headingSearchIndex`) the context would be much clearer.

E.g.
> Level 6 header **(outside headingSearchIndex)** with index attribute...